### DISCUSSION TITLE
Thankyou!

### DISCUSSION THREAD

**Discussion by goodstout** [JoseConseco/GoB#476] [General]
I just wanted to say thanks to the folks who picked this up and started cleaning it up after the Blender 4.0 update.

**Comment by Laticis** [comment]
I would also like to say Thank you! - very much appreciated. Working with ZBrush 2022 & Blender 4.3

**Comment by Snee3D** [comment]
Massive massive thank you. using Zbrush 2022 with Blender 4.5.3 LTS
